lib/builtins/os_version_check.c

```diff
@@ -77,6 +90,13 @@ typedef void (*CFReleaseFuncTy)(CFTypeRef);
 /* Find and parse the SystemVersion.plist file. */
 static void parseSystemVersionPList(void *Unused) {
```

Comment:
@hyp this needs to be updated to `initializeAvailabilityCheck`
```suggestion
static void initializeAvailabilityCheck(void *Unused) {
```